kilkari/src/main/java/org/motechproject/nms/kilkari/domain/CallRetry.java

```diff
@@ -57,6 +66,7 @@ public CallRetry(String subscriptionId, Long msisdn, //NO CHECKSTYLE More than 7
         this.languageLocationCode = languageLocationCode;
         this.circle = circle;
         this.subscriptionOrigin = subscriptionOrigin;
+        this.timestamp = timestamp;
```

Comment:
if we are adding this field, maybe we should also just remove the dayOfTheWeek right now?